GVFS/GVFS.Platform.Mac/MacPlatform.cs

```diff
@@ -1,157 +1,22 @@
 ﻿using GVFS.Common;
 using GVFS.Common.FileSystem;
-using GVFS.Common.Git;
-using GVFS.Common.Tracing;
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.IO.Pipes;
 
 namespace GVFS.Platform.Mac
 {
-    public partial class MacPlatform : GVFSPlatform
+    public class MacPlatform : POSIX.POSIXPlatform

```

Comment:
Minor: Change `POSIX.POSIXPlatform` to `POSIXPlatform` and add a `using` for the POSIX namespace.